app/controllers/books_controller.rb

```diff
@@ -9,6 +9,9 @@ def index
   end
 
   def show
+    @book = Rails.cache.fetch("book-#{params[:id]}", expires_in: 1.minute) do
+              Book.find(params[:id])
+            end
```

Comment:
this is not a good idea to cache every book show for a minute.
You should not use cache here.
it can be used when your data remains unchanged or changes after long time. 
such as we can put our top 12 new books in cache for a day or week may be.